### PR TITLE
gives the crew more to do than shoot ships get shot move paper

### PR DESCRIPTION
**Pull request by TMTIME**
instead gives them new missions to sit in a system and shoot ships and move lots of paper!

:cl: TMTIME
add: Trade mission
add: Hold system mission
/:cl:

Adds a small amount of variety to a round with new objectives.

The first is simple. Raise some money. Somewhat easy if they sell tech disks.

The second is wave defence in syndicate space. Eliminate a set number of waves of increasing size.

### CODE REVIEW

File: code/controllers/subsystem/ships.dm [hunk 1]
```diff
@@ -297,6 +297,9 @@ SUBSYSTEM_DEF(ship)
 	for(var/datum/objective/ftl/killships/O in SSstarmap.ship_objectives)
 		if(S.faction == O.faction)
 			O.ships_killed++
+	for(var/datum/objective/ftl/hold_system/O_h in SSstarmap.ship_objectives)
+		if(SSstarmap.current_system == O_h.target_system && O_h.wave_active)
+			O_h.ships_remaining --
```
Comment (Qustinnus) [inline code/controllers/subsystem/ships.dm]: why the space between the var and the decrement

Comment (Qustinnus) [reply code/controllers/subsystem/ships.dm]: i mean if it works it works just wanna make sure

Comment (TMTIME) [reply code/controllers/subsystem/ships.dm]: Old habit I guess? commit up to remove it

**Comment by Qustinnus** [comment]
theres more of them 👀

**Pull request merged**